src/main/java/org/opensearch/geospatial/ip2geo/jobscheduler/DatasourceExtension.java

```diff
@@ -21,6 +23,20 @@ public class DatasourceExtension implements JobSchedulerExtension {
      * Job index name for a datasource
      */
     public static final String JOB_INDEX_NAME = ".scheduler_geospatial_ip2geo_datasource";
+    /**
+     * Job index setting
+     *
+     * We want it to be single shard so that job can be run only in a single node by job scheduler.
+     * We want it to expand to all replicas so that querying to this index can be done locally to reduce latency.
+     */
+    public static final Map<String, Object> INDEX_SETTING = Map.of(
+        "index.number_of_shards",
+        1,
+        "index.auto_expand_replicas",
+        "0-all",
+        "index.hidden",
+        true
+    );
```

Comment:
I think we need constants for keys, previously each separate variable as like such constant, but we lost it with the map

Reply:
INDEX_SETTING itself is a constant so I didn't make value in it as another constant.